snap/local/get_sbat_var_previous.py

```diff
@@ -0,0 +1,31 @@
+import sys
```

Comment:
This file is missing a shebang and header.

I do think even with moving parsing into fwupd you can keep this script in contrib as it could be useful for debugging purposes.

Reply:
File was removed